esp-wifi/src/wifi/mod.rs

```diff
@@ -97,31 +98,72 @@ impl WifiMode {
     }
 }
 
+const DATA_FRAMES_MAX_COUNT: usize = RX_QUEUE_SIZE + RX_QUEUE_SIZE;
```

Comment:
@bjoernQ Is this a typo (`RX_QUEUE_SIZE + TX_QUEUE_SIZE`) or is this intentionally 2x?

Reply:
looks like a typo, true. should be RX + TX queue size 👍